app/jobs/itms_job.rb

```diff
@@ -28,27 +28,34 @@ def self.get(url)
       raise DownloadError, "Unexpected response code: #{response.code} for url: #{url}" if response.code != 200
       response
     end
+
+    private
+
+    def self.format_query(params)
+      params.reject! { |_, v| v.blank? }
+      "?#{params.to_query}" if params.present?
```

Comment:
toto nebude dobre - co ak `href` uz obsahuje query?

hint: `URI(...).tap { |url| url.query = [url.query, ...].compact.join('&') }.to_s` a mozno sa to da aj lepsie